packages/near-membrane-node/src/node-realm.ts

```diff
@@ -12,20 +12,20 @@ import { runInNewContext } from 'vm';
 
 interface NodeEnvironmentOptions {
     distortionCallback?: DistortionCallback;
-    endowments?: object;
-    globalThis: typeof globalThis;
+    endowments: object;
+    globalThis?: typeof globalThis;
```

Comment:
Why is `globalThis` now optional? What happens if it's not provided?

Reply:
It was optional, but it seems the types were not correct. If you look at the code, it was always optional.